docker/cloudbuild.yaml

```
@@ -2,9 +2,9 @@
 # $ gcloud builds submit
 
 substitutions:
-  _DOCKER_19: DOCKER_VERSION=5:19.03.9~3-0~ubuntu-focal
-  _DOCKER_20: DOCKER_VERSION=5:20.10.14~3-0~ubuntu-focal
-  _DOCKER_24: DOCKER_VERSION=5:24.0.6-1~ubuntu.20.04~focal
+  _DOCKER_19: DOCKER_VERSION=5:19.03.15~3-0~ubuntu-focal
+  _DOCKER_20: DOCKER_VERSION=5:20.10.24~3-0~ubuntu-focal
+  _DOCKER_24: DOCKER_VERSION=5:24.0.7-1~ubuntu.20.04~focal
```

Looks like 24.0.9 is available

```bash
apt list -a docker-ce | grep 5.24.0.

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

docker-ce/focal 5:24.0.9-1~ubuntu.20.04~focal amd64
docker-ce/focal 5:24.0.8-1~ubuntu.20.04~focal amd64
docker-ce/focal 5:24.0.7-1~ubuntu.20.04~focal amd64
docker-ce/focal 5:24.0.6-1~ubuntu.20.04~focal amd64
docker-ce/focal 5:24.0.5-1~ubuntu.20.04~focal amd64
docker-ce/focal 5:24.0.4-1~ubuntu.20.04~focal amd64
docker-ce/focal 5:24.0.3-1~ubuntu.20.04~focal amd64
docker-ce/focal 5:24.0.2-1~ubuntu.20.04~focal amd64
docker-ce/focal 5:24.0.1-1~ubuntu.20.04~focal amd64
docker-ce/focal 5:24.0.0-1~ubuntu.20.04~focal amd64
```

```suggestion
  _DOCKER_24: DOCKER_VERSION=5:24.0.9-1~ubuntu.20.04~focal
```

---

Thanks for looking into that, I created another PR (had to delete my fork for some other reason)
#980